script/playbackstrategy/modifiers/html5.js

```diff
@@ -522,7 +522,7 @@ define(
         var clampedTime = getClampedTime(seconds);
         var range = getSeekableRange();
         if (clampedTime !== seconds) {
-          logger.debug('playFrom ' + seconds + ' clamped to ' + clampedTime + ' - seekable range is { start: ' + range.start + ', end: ' + range.end + ' }');
```

Comment:
Any reason why this changed?

Reply:
Matches up with tests, changed this one because its quicker